test-app/app/build.gradle

```diff
@@ -222,6 +227,10 @@ dependencies {
 ///////////////////////////// CONFIGURATION PHASE //////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////////
 
+task exportConfigurationPaths {
+    getAppResourcesDirectory()
```

Comment:
Can we move the invocation of getAppResourcesDirectory to applyAppGradleConfiguration?

Reply:
There already is invocation of `getAppResourcesDirectory` in `applyAppGradleConfiguration`, unless that's too late for whatever purposes you need the externalized properties for, then the task is redundant.